plugins/modules/wafv2_web_acl.py

```diff
@@ -341,7 +341,6 @@ def update(self, default_action, description, rules, sampled_requests, cloudwatc
             'Scope': self.scope,
             'Id': self.id,
             'DefaultAction': default_action,
-            'Description': description,
             'Rules': rules,
```

Comment:
`rules` shouldnt be required based off of the docs - can you treat `rules` the same way you treated `description` and remove `rules` from being required when `state=present`?

Reply:
I agree on that!

Reply:
Due to the way that the WAFv2 API works, this would require some very non-trival changes (a near total rewrite level of non-trivial).

No.